css/build.css

```
@@ -120,3 +121,89 @@
     padding: 40px;
   }
 }
+
+[data-notification-inbox-1-0-0-id] .switch-toggle-holder {
+  display: flex;
+  align-items: center;
+  float: left;
+  margin-right: 2px;
+  padding: 0 0.125em 20px;
+  line-height: 1.5;
+  height: 39px;
+}
+
+[data-notification-inbox-1-0-0-id] .switch-toggle-holder span {
+  cursor: pointer;
+}
+
+[data-notification-inbox-1-0-0-id] .switch {
+  position: relative;
+  display: inline-block;
+  width: 36px;
+  height: 18px;
+  margin: 0 8px;
+}
+
+[data-notification-inbox-1-0-0-id] .switch input {
+  display: none;
+}
+
+[data-notification-inbox-1-0-0-id] .slider {
+  position: absolute;
+  cursor: pointer;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: #dcdae0;
+  -webkit-transition: 0.25s;
+  transition: 0.25s;
+}
+
+[data-notification-inbox-1-0-0-id] .slider::before {
+  position: absolute;
+  content: "";
+  height: 14px;
+  width: 14px;
+  left: 4px;
+  bottom: 2px;
+  background-color: #fff;
+  -webkit-transition: 0.25s;
+  transition: 0.25s;
+}
+
+[data-notification-inbox-1-0-0-id] .slider.round {
+  border-radius: 16px;
+}
+
+[data-notification-inbox-1-0-0-id] .slider.round::before {
+  border-radius: 50%;
+}
+
+[data-notification-inbox-1-0-0-id] input:checked + .slider {
+  background-color: #000;
+}
+
+[data-notification-inbox-1-0-0-id] input:focus + .slider {
+  box-shadow: 0 0 1px #2196f3;
+}
+
+[data-notification-inbox-1-0-0-id] input:checked + .slider::before {
+  -webkit-transform: translateX(14px);
+  -ms-transform: translateX(14px);
+  transform: translateX(14px);
+}
+
+[data-notification-inbox-1-0-0-id] .unread {
+  width: 100%;
+  float: left;
+  padding-left: 10px;
+}
+
+[data-notification-inbox-1-0-0-id] .unread-label {
```

See section 10.2 in https://docs.google.com/document/d/1XPlQHPbqp3rGGTekcC05bKtwKDDjrU2RMJ4VIHLXp64/edit#

```suggestion
[data-notification-inbox-1-0-0-id] .label-unread {
```

---

Did the changes suggested by tony. Also gone through the docs